### ISSUE TITLE
[Feature Request] Return Matching Blocks

### ISSUE THREAD

**Issue by rapatel0** [jplattel/obsidian-query-language#9]
It would be great if Oql could return the text of a matching block from a query. Specifying `block:` or `line:` like the Obisidian search API does.

**Comment by jplattel** [comment]
Yeah, that would really nice to have. But currently the way Fuse indexes files doesn't allow for any understanding of the contents.. So building that would be hard to do right now. I do like the idea, so if there's any change of things improving on this I will definitely keep this in mind :)
